corehq/apps/accounting/tests/test_forms.py

```diff
@@ -404,3 +407,36 @@ def test_num_form_submitting_mobile_workers(self):
             domain=self.domain.name, record_date=self.statement_end
         )
         self.assertEqual(user_history.num_users, num_users)
+
+
+class TestPlanContactForm(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.domain = generator.arbitrary_domain()
+        self.web_user = generator.arbitrary_user(self.domain.name, is_webuser=True)
+
+    def tearDown(self):
+        self.domain.delete()
+        super().tearDown()
```

Comment:
`self.addCleanup` is preferred to `tearDown` because it will be run even if `setUp` fails.
```suggestion
        self.addCleanup(self.domain.delete)
        self.web_user = generator.arbitrary_user(self.domain.name, is_webuser=True)
```